cmd/lambda/publish_lease_events/main.go

```diff
@@ -113,18 +113,22 @@ func handleRecord(input *handleRecordInput) error {
 		if didBecomeInactive {
 			// Before adding the account to any queues, make sure the account is
 			// updated to NotReady state.
-			_, err = input.dbSvc.TransitionAccountStatus(lease.AccountID, db.Leased, db.NotReady)
+			acct, err := input.dbSvc.TransitionAccountStatus(lease.AccountID, db.Leased, db.NotReady)
 			if err != nil {
-				log.Printf("ERROR: Failed to mark AccountStatus=NotReady for %s, after lease for %s became inactive",
```

Comment:
?

Reply:
@eschwartz are you "?" why the logging statement was removed? @kddejong, does seem like that information might be useful for debugging.

Reply:
Partially.  It will already print the error but juust not there.  So in a way this print is duplicative.  Additionally this error handling doesn't return err it continues on... silently continues if you will.